iis-wf/iis-wf-import/src/main/java/eu/dnetlib/iis/wf/importer/infospace/converter/DocumentMetadataConverter.java

```diff
@@ -202,18 +201,18 @@ private void handlePublicationType(List<Instance> instanceList, DocumentMetadata
      */
     private DocumentMetadata.Builder handleAdditionalIds(OafEntity oafEntity, DocumentMetadata.Builder metaBuilder) {
         // setting additional identifiers
-        Map<CharSequence, CharSequence> additionalIds = new HashMap<CharSequence, CharSequence>();
-        if (CollectionUtils.isNotEmpty(oafEntity.getPidList())) {
-            for (StructuredProperty currentPid : oafEntity.getPidList()) {
+        if (CollectionUtils.isNotEmpty(oafEntity.getPid())) {
+            Map<CharSequence, CharSequence> additionalIds = new HashMap<CharSequence, CharSequence>();    
+            for (StructuredProperty currentPid : oafEntity.getPid()) {
                 if (StringUtils.isNotBlank(currentPid.getQualifier().getClassid()) 
                         && StringUtils.isNotBlank(currentPid.getValue()) 
                         && fieldApprover.approve(currentPid.getDataInfo())) {
                     additionalIds.put(currentPid.getQualifier().getClassid(), currentPid.getValue());
                 }
             }
```

Comment:
For the record: one modification is required:
```java
.collect(Collectors.toMap(AbstractMap.SimpleEntry::getKey, AbstractMap.SimpleEntry::getValue, (val1, val2) -> val1));
```
instead of:
```java
.collect(Collectors.toMap(AbstractMap.SimpleEntry::getKey, AbstractMap.SimpleEntry::getValue));
```
otherwise we end up with `java.lang.IllegalStateException: Duplicate key`. 

I remember we've had similar problem some time ago when introducing streaming in `DatasetMetadataConverter` and this was our solution back then.